### PR TITLE
Add Style

### PR DESCRIPTION
**Pull request by danmainah**
### Bookstore: styling### 

**In this milestone I added styling to my design and make sure it matches the  Zeplin's design**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,25 +1,20 @@
...
+  width: 85%;
+  margin: auto;
+  background: #fafafa;
 }
 
-.navbar a {
-  color: #25649e;
+ul {
+  background: #fafafa;
 }
```
Comment (DeliceLydia) [inline src/App.css]: - [x] As the requirements say you should follow the design on [Zeplin]() that means colors, layouts etc.
- [x] They are features that were not implemented yet you should add them as placeholders like comments, edit, update progress button etc.
refer to this:
zp

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @danmainah, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.css (hunk 1 above)
Comment (danmainah) [reply src/App.css]: Thank you for the review I have implemented the requested changes.

**Review by BrianSammit — APPROVED** [review]
# STATUS: APROVED :green_circle: :clap: 

Hello @danmainah, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

200w


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**